### DISCUSSION TITLE
Concerns ... BattleScribe file format is proprietary?

### DISCUSSION THREAD

**Discussion by dstidham63** [BSData/phalanx#51] [General]
Not to be a wet noodle, but isn't building a "new" tool using the existing BattleScribe file format asking for trouble from the author of BattleScribe?  Isn't the file format a proprietary structure that the author could claim is copyright infringement?

**Comment by AbonShell** [comment]
The Dev for battlescribe doesn't own the data created with, or by anyone that uses the tools provided.

**Reply by dstidham63** [reply]
I'm not referring to the raw data.  I am referring to the actual data structure that the BattleScribe author devised to store the data elements.  The BattleScribe owner does "own" the data structures ...

**Reply by amis92** [reply]
Please share your resources backing the claim that you can copyright data formats.

It's first time I've seen such a claim.

If you do not have any specific references to substantiate your claim, I suggest we close this issue with an agreement that this is a non issue 😅

**Reply by dstidham63** [reply]
My concern is with the proprietary, as in unique and secret, nature of the BattleScribe table structure.  My use of "copyright infringement" may have been incorrect.

While I doubt that the author of BattleScribe applied for a software patent or a formal copyright, but that does not mean the author couldn't reappear on the scene and take issue with, and create hardship for, this project and could have legal grounds based up the documented age of BattleScribe as an implied patent or copyright.

Has anyone from this project had any contact with the author of BattleScribe to obtain permission or blessing to proceed?  

Has anyone considered insulating this initiative through the separate development of a new data model, and discrete standalone conversion tool that reads the BattleScribe files and creates new files that can be used by Phalanx?

I'm not trying to be a pain in the keyboard, so to speak, but being in the software industry for over 3 decades, I have seen a great deal of things transpire that fall within this scenario.  Think back to the days of olde when Microsoft Excel files (XLS) were _proprietary_.  No other company was able to legally open and read those files for their own purpose outside of Excel.  Microsoft has gotten softhearted and made the XLS format open specification format, and now everyone can freely develop something to read them.

**Reply by AbonShell** [reply]
The .cat files are just .xml with a different file extension. .catz are just zipped .cat files.

**Reply by amis92** [reply]
@dstidham63 of course. Thanks for your concerns.

However we're not expecting any problems here, and with the original app author gone for over 2 years making BattleScribe abandonware at that point, we on the other hand have a convincing and substantiated reason to produce a compatible program.

Succintly, we really don't care what the BS author may or may not have to say on this matter. As the leading contributor located in Poland and no other entity to sue, he'd have to sue me in here. He's a single man company. It doesn't seem to have a benefit to him. And such a move would create a huge backslash from community, effectively rendering his company unprofitable, as we've already seen a backlash on iOS AppStore dropoff for *just a couple of hours*. :)

**Reply by dstidham63** [reply]
Thank you ... you have a plan in hand, @amis92.  That's all I was asking about.
